addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/handler/PentairSerialBridgeHandler.java

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+/* PentairSerialBridgeHandler.java
```

Comment:
I think this additional header should be removed, you have a comment further below anyhow.